lib/config/index.ts

```diff
@@ -25,7 +26,7 @@ export function getManagerConfig(
   }
   // TODO: fix types #7154
   managerConfig = mergeChildConfig(managerConfig, config[manager] as any);
-  for (const i of getManagerList()) {
+  for (const i of [...getManagerList(), ...getCustomManagerList()]) {
```

Comment:
can the `getManagerList` return all managers? so we don't have to clone e arrays? (perf)

if not create a new static list and function like `getAllManagersList()`